cmake/FindMatlab.cmake

```diff
@@ -29,6 +33,17 @@ The module supports the following components:
 * ``MCC_COMPILER`` the MCC compiler, included with the Matlab Compiler add-on.
 * ``SIMULINK`` the Simulink environment.
 
+.. versionadded:: 3.7
+  Added the ``MAT_LIBRARY`` component.
+
+.. versionadded:: 3.13
+  Added the ``ENGINE_LIBRARY``, ``DATAARRAY_LIBRARY`` and ``MCC_COMPILER``
+  components.
+
+.. versionchanged:: 3.14
```

Comment:
what do these `versionchanged` tags imply?

Reply:
This is something that is part of the imported CMake file. I think it is used internally by CMake, but I don't have other details.